amarachukwu-agbo/hello-books
# #157772370 Add tests for server-side code

#### What does this PR do?

Add tests for the backend code

#### Description of Task to be completed?

- Write tests for books controller
- Write tests for users controller

#### How should this be manually tested?

Clone the repo and install application dependencies. Run command  `npm test`. All the tests should run successfully.

#### Any background context you want to provide?

N/A

#### What are the relevant pivotal tracker stories?

#157772370

#### Screenshots (if appropriate)

N/A

#### Questions: N/A
